tests/chainerx_tests/unit_tests/routines_tests/test_indexing.py

```diff
@@ -368,3 +390,25 @@ def test_where_scalar_scalar(xp, cond_shape, cond_dtype, in_types, out_dtype):
     y = y_type(2)
     out = xp.where(cond, x, y)
     return dtype_utils.cast_if_numpy_array(xp, out, out_dtype)
+
+
+# TODO(aksub99): Add cuda tests after Cumsum cuda kernel is implemented
+@op_utils.op_test(['native:0'])
+@chainer.testing.parameterize_pytest('input', _nonzero_params)
+class TestNonzero(op_utils.NumpyOpTest):
+
+    check_numpy_strides_compliance = False
+    skip_backward_test = True
+    skip_double_backward_test = True
```

Comment:
Could you explain why did you skip these tests?

Reply:
Now that you mention it, I don't think `check_numpy_strides_compliance` should be set to False. I had assumed that it may have been for 0-dim inputs. But the following error seems to occut for 2-dim cases if it is set to true.
```
chainer.testing.function_link.FunctionTestError: Strides do not match with NumPy outputs.
E       Expected shapes and dtypes: (6,):int64, (6,):int64
E       Actual shapes and dtypes:   (6,):int64, (6,):int64
E       Expected strides: (16,), (16,)
E       Actual strides:   (8,), (8,)
```
As for the backward tests, https://github.com/HIPS/autograd/blob/6e5ec96993bd7136c8d96cc6bfb11dccd04490aa/autograd/numpy/numpy_vjps.py#L11 mentions that `nonzero` shouldn't have a backward implementation.

Reply:
Whether ChainerX support backward or not is not directly related to whether other flameworks support it. Actually, ChainerX has already some differentiable functions listed here as nograd_functions.
Please support backward and double backward.